docs/spec/functional_spec.rst

```diff
@@ -577,14 +577,12 @@ Check whether a burrow can be liquidated
 +---------------+-----------------------+-------------------------------------------------------------------------+
 
 
-Minimum bid for the current liquidation auction (if exists)
------------------------------------------------------------
+Get details on the current liquidation auction
+----------------------------------------------
 
-Returns a pair of the unique identifier for the current auction and the minimum
-amount of kit that one could currently bid in said auction. Fails if there is
-no open auction.
+Fails if there is currently no liquidation auction.
 
-``current_liquidation_auction_minimum_bid : unit -> pair nat nat``
+``current_liquidation_auction_details: unit -> view_current_liquidation_auction_details_result``
```

Comment:
@dorranh In the functional spec we've been using Michelson types (that's what runs on the chain so it's the Michelson code that needs a spec). Since we are not using `[@layout:comb]`, `view_current_liquidation_auction_details_result` is most likely represented in Michelson as
```
(pair nat (pair nat (pair nat (pair (option (pair address nat)) (pair (option int) (option int))))))
```
 so this should probably be:
```
``current_liquidation_auction_details: unit -> (pair nat (pair nat (pair nat (pair (option (pair address nat)) (pair (option int) (option int))))))``
```

Reply:
Ah good catch. I'll submit an update.